docs/modules/datasets/tdc.rst

```diff
@@ -0,0 +1,79 @@
+=================================
```

Comment:
Number of bars should be aligned with text length, e.g. "TDC", "ADME dataset loaders"